content/posts/networkx/GSoC-2022/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Hello World"
+date: 2022-06-09
+draft: false
+description: "Getting started"
+tags: ["gsoc", "networkx"]
+displayInList: true
+author: ["kpetridis24"]
+
+resources:
+  - name: featuredImage
+    src: "GSoC_logo.png"
```

Comment:
@mjschwenne had created a version of the logo that has both the networkx & GSoC logos in a single png - I think we should use that here as well --- esp. since the blog has posts from GSoC participants from other projects.

The easiest way to do this is probably to copy the `mjschwenne_GSoC.png` file over from one of the other directories (you can rename it in this folder if you'd like). Even better might be to try to link out to it, but we can always worry about that in a followup PR.

Reply:
I actually got the idea from the other GSoC posts like Aitik Gupta's Matplotlib posts. All I did was fire up GIMP and replace the Matplotlib logo with a NetworkX logo :smile:.